tasks/manage_lvm_vg.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: LVG | Show current LVM volume groups
+  debug: msg="Running on {{ item.vgname }} because {{ item.state }}"
+  with_items: "{{ lvm_groups }}"
+
+- name: LVG | Creating new LVM volume group
+  lvg:
+    vg: "{{ lvm_vgname }}"
+    pvs:
+      - "{{ lvm_disk }}"
+    state: present
+  with_items: "{{ lvm_groups }}"
+  when:
+    - item.state|default("present") == "present"
+    - item.vgname == lvm_vgname
+    - lvm_action == "add_vg"
+
+- name: LVG | Removing LVM volume group
+  debug: msg="You requested *** DELETION *** of volume group {{ lvm_vgname }}. Too dangerous, do it manually please."
```

Comment:
safer be default indeed :)

(maybe it could be useful to have an extra var to force the deletion `lvm_force_deletion=true`)

Reply:
The working piece of code is commented here: https://github.com/peopledoc/ansible-lvm/blob/30fade8570927958617668ce9ffc543ad754255d/tasks/manage_lvm_vg.yml#L26-L34

Because it's a very rare need, I prefer avoid a single extra var that can be too easily recalled by a bash history search :/

But I'm open to a better suggestion.

Reply:
indeed no need